TESTING.md

```diff
@@ -493,4 +492,53 @@ available. Click here to open the feedback.** notification is shown.
 
 **13.3** Type the characters *fl* on your keyboard.
 
-**13.4** ASSERTION: **Week 5** is automatically expanded, **Assignment 4 (FlappyBug 15)** is automatically selected, and the substring **Fl** is highlighted.
+**13.4** ASSERTION: **Week 5** is automatically expanded, **Assignment 4 (FlappyBug 15)** is automatically selected, and
+the substring **Fl** is highlighted.
+
+### 14 Downloading student submissions
+
+**14.1** Enable Assistant Mode from the A+ menu.
+
+**14.2** ASSERTION: The student selection and submission downloading buttons appear above the assignments.
+
+**14.3** Click the student selection button.
+
+**14.4** ASSERTION: After loading, the student selection dialog opens.
+
+**14.5** ASSERTION: The dialog lists students with their names and student IDs.
+
+**14.6** Select a student with submissions by double-clicking the student.
```

Comment:
Remark: Should we use some specific student's data for testing? Test user?

Reply:
Maybe I could add a test user when O1 2021 starts